### PR TITLE
Netstat tips

### PR DESCRIPTION
**Pull request by GustavoKatel**
Adding the netstat command to the list. It's useful to find which process is using a specific port and find possible malicious ip's connected to the server

### CODE REVIEW

**Comment by gustavosobral** [comment]
Hey @GustavoKatel, thanks for the contribution! Netstat is really useful for find possible vulnerabilities. Could you just change the bash indicator from `#` to `$` instead?

**Comment by GustavoKatel** [comment]
The `p` parameter requires `su` to show all the processes. It works without it, but it could omit information.

File: README.md [hunk 1]
```diff
@@ -25,6 +25,23 @@ My common Iptables configuration:
 
 ![Iptables](assets/iptables.png)
 
+## Netstat
+Print network connections
+
+List tcp connections (t), without resolving ip addresses (n) and showing the process pid (p)
+```
+# netstat -tnp
```
Comment (gustavosobral) [inline README.md]: I was talking about this character `#`, indicating the bash command line. I'm using the `$` instead for this purpose. :smile:

**Comment by gustavosobral** [comment]
:tada:

**Pull request merged**